x-pack/plugins/ml/public/application/components/anomalies_table/links_menu.tsx

```diff
@@ -50,6 +54,24 @@ import { useMlKibana } from '../../contexts/kibana';
 import { getFieldTypeFromMapping } from '../../services/mapping_service';
 import type { AnomaliesTableRecord } from '../../../../common/types/anomalies';
 
+function getQueryStringForInfluencers(
```

Comment:
I added some comments above, but this function might be better as a self contained reduce.
Something like this (this is untested :) )

```
function getQueryStringForInfluencers(
  influencers: AnomaliesTableRecord['influencers'] = [],
  entityName?: string
) {
  return influencers
    .reduce<string[]>((acc, influencer) => {
      Object.entries(influencer)
        .filter(([name]) => name !== entityName)
        .forEach(([name, inf]) => acc.push(`${name}: ${inf}`));
      return acc;
    }, [])
    .join(' or ');
}
```

Reply:
Updated and tested in 67868c24accc7e8dfaf3fa550c946f9ee96aa032